### DISCUSSION TITLE
Is it just me, or is Streamhost not working anymore?

### DISCUSSION THREAD

**Discussion by yumiyacha1** [iptv-org/community#162] [Q&A] [answered]
When I accessed their website, it redirected me to a website with "ww1.streamhost.cc" containing something about "related searches". This makes me believe that this is the reason why some channels in the Philippines playlist don't work, or at least "couldn't be read"
![image](https://user-images.githubusercontent.com/82864438/153720466-7509efbd-591a-4445-b141-53c8f97ea293.png)

**Comment by BellezaEmporium** [comment, marked as answer]
Most certainly domain wasn't renewed or devs ceased to use this domain.
